docs/README.md

```diff
@@ -6,4 +6,4 @@ This plugin supports a two-way integration between Mattermost and Jira. For a st
 
 This plugin supports Jira Core and Jira Software products for Server, Data Center, and Cloud platforms. It has been tested with versions 7 and 8.
 
-**Support for multiple Jira instances is supported from Jira 3.0 and configured using [Administrator Slash Commands](https://mattermost.gitbook.io/plugin-jira/admininstrator-guide/administrator-slash-commands). Available for Mattermost Enterprise E20.**
+**Support for multiple Jira instances is supported from Jira 3.0 for Mattermost Enterprise Editions and configured using [Administrator Slash Commands](https://mattermost.gitbook.io/plugin-jira/admininstrator-guide/administrator-slash-commands).**
```

Comment:
An E20 license is required to use the feature, i.e. an instance with an E10 enterprise license won't work. Can we keep the part about E20 in to be clear?